schemas/score-v1b1.json

```diff
@@ -228,28 +254,50 @@
             },
             "oneOf": [
               {
+<<<<<<< HEAD
```

Comment:
More here?

Reply:
fixed up! yup looks like some undetected merge conflict. I've re-copied the schema in from source.